dataland-frontend/src/components/resources/frameworkDataSearch/TwoLayerDataTable.vue

```diff
@@ -70,7 +70,28 @@
                   <em class="material-icons" aria-hidden="true" title=""> dataset </em>
                 </a>
 
-                <span v-else> {{ slotProps.data.content[reportingPeriodWithDataId.dataId][0] }} </span>
```

Comment:
Why was the content of the span deleted? Do we no longer need this? Do we now need this empty <span>?

Reply:
We removed it because we don't want to display the first value of the array as a standalone value. 
(length > 1)
We want the array to be displayed as a modal even if it is a single-element array. 
(length >= 1)
And the empty span is used when the array is completely empty
(length == 0)

Reply:
ok